flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -251,4 +252,20 @@ class AndroidNotificationDetails {
   /// For a list of a values, refer to the documented constants prefixed with "FLAG_" (without the quotes) at https://developer.android.com/reference/android/app/Notification.html#constants_1.
   /// For example, use a value of 4 to allow the audio to repeat as documented at https://developer.android.com/reference/android/app/Notification.html#FLAG_INSISTEN
   final Int32List additionalFlags;
+
+  /// This provides some additional information that is displayed in the
+  /// notification. No guarantees are given where exactly it is displayed.
+  ///
+  /// This information should only be provided if it provides an essential
+  /// benefit to the understanding of the notification. The more text you
+  /// provide the less readable it becomes. For example, an email client should
+  /// only provide the account name here if more than one email account has
+  /// been added.
+  ///
+  /// As of Build.VERSION_CODES.N this information is displayed in the
```

Comment:
`Build.VERSION_CODES.N` is a constant within the Android SDK, could replace this with the Android version number as that's the convention that's been used within the plugin. This was done as Flutter developers may not have any familiarity with Android development to know the version codes or API levels. Outside of software development Android versions are now being referred to by numbers as well, which in this case is 7.0